hack/verify-all.sh

```diff
@@ -27,14 +26,19 @@ res=0
 
 # run all verify scripts
 "${REPO_ROOT}"/hack/verify-govet.sh || res=1
+
+cd "${REPO_ROOT}"
 "${REPO_ROOT}"/hack/verify-gofmt.sh || res=1
+
+cd "${REPO_ROOT}"
 "${REPO_ROOT}"/hack/verify-golint.sh || res=1
```

Comment:
bikeshed: the script also has `cd "${REPO_ROOT}"`? just for consistency?

Reply:
sure, we'll CD twice in a row though :-)

Reply:
actually restructured so it's script -> cd back